catalyst_voices/packages/internal/catalyst_voices_view_models/lib/src/proposal/guidance/guidance.dart

```diff
@@ -0,0 +1,41 @@
+import 'package:catalyst_voices_view_models/src/proposal/guidance/guidance_type.dart';
+import 'package:equatable/equatable.dart';
+
+final class Guidance extends Equatable {
+  final String title;
+  final String description;
+  final GuidanceType type;
+  final int? weight; // This represents how important the guidance is in
+  //specific [GuidanceType].
+
+  const Guidance({
+    required this.title,
+    required this.description,
+    required this.type,
+    this.weight,
+  });
+
+  String get weightText => weight?.toString() ?? '';
+
+  @override
+  List<Object?> get props => [
+        title,
+        description,
+        type,
+      ];
+}
+
+extension GuidanceExt on List<Guidance> {
+  List<Guidance> sortedByWeight() {
```

Comment:
consider making `Guidance` implement `Comparable` and move it logic there.

Reply:
Great suggestion!